Classes/Emogrifier.php

```diff
@@ -195,7 +195,50 @@ public function emogrify()
             throw new \BadMethodCallException('Please set some HTML first before calling emogrify.', 1390393096);
         }
 
-        $xmlDocument = $this->createXmlDocument();
+        $xmlDocument = $this->process($this->createXmlDocument());
+        
+        return $xmlDocument->saveHTML();
+    }
+                
+    /**
+     * Applies the CSS you submit to the HTML you submit and returns only the
+     * HTML content within the <body> tag.
+     *
+     * This method places the CSS inline.
+     *
+     * @return string
+     *
+     * @throws \BadMethodCallException
+     */
+    public function emogrifyBodyContent()
+    {
+        if ($this->html === '') {
+            throw new \BadMethodCallException('Please set some HTML first before calling emogrify.', 1390393096);
+        }
+
+        $xmlDocument = $this->process($this->createXmlDocument());
+        
+        $innerDocument = new \DOMDocument();
+        foreach ($xmlDocument->documentElement->getElementsByTagName('body')->item(0)->childNodes as $childNode) {
+            $innerDocument->appendChild($innerDocument->importNode($childNode, true));
+        }
+        
+        return $innerDocument->saveHTML();
+    }
+    
+    /**
+     * Applies the submitted CSS to the submitted HTML.
+     *
+     * This method places the CSS inline.
+     *
+     * @param \DOMDocument $xmlDocument
+     *
+     * @return \DOMDocument
+     *
+     * @throws \BadMethodCallException
```

Comment:
This annotation is not true as this method does not throw the exception. Could you please remove it?